### PR TITLE
feature/responsive-nav-last-assets

### PR DESCRIPTION
**Pull request by Rallen13**
#19 
#20 
#24

** Describe the functionality to be tested: **

- assets: SVG logo, homepage hero image
- nav: fully responsive so shouldn't have issues with alignment anymore, added in SVG logo
- footer: fully responsive so shouldn't have issues with alignment anymore, added in SVG logo, added in github links for team members

** Giphy of how you feel submitting this PR: **

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -15,6 +15,10 @@
       href="https://fonts.googleapis.com/css2?family=Rubik:wght@400;700&display=swap"
       rel="stylesheet"
     />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
```
Comment (irmakerdem) [inline public/index.html:20]: Curious to find out if this to fill/thicken the outlines of icons

File: src/Components/Footer/Footer.scss [hunk 2]
```diff
@@ -1,9 +1,17 @@
...
   background-color: #3c3525;
   padding: 1.5rem;
   display: flex;
   flex-direction: row;
   justify-content: space-between;
+  flex-wrap: wrap;
+
+  .logo-img {
+    height: 4rem;
+    filter: invert(1);
```
Comment (lkarpins) [inline src/Components/Footer/Footer.scss:11]: That logo warms my heart! ❤️

**Review by irmakerdem — APPROVED** [review]
Nice job, Rachel! Excited to see the cute contact icons be clickable!

File: public/index.html (hunk 1 above)
Comment (Rallen13) [reply public/index.html:20]: So this is the stylesheet that allows us to use material UI icons with classnames on <span> tags, you can see an example of one in the footer after our names. This is the line Google gives you to add.

**Review by lkarpins — APPROVED** [review]
Great work, Rachel!!

File: src/Components/Footer/Footer.scss (hunk 2 above)
Comment (Rallen13) [reply src/Components/Footer/Footer.scss:11]: That filter: invert is sweet!! Essentially all that does is change the logo from black to white! I found it in a stack overflow article I can share with yall tomorrow.

**Review by catlynbowles — APPROVED** [review]

**Pull request merged**